src/conversion/docs/readme.md

```diff
@@ -0,0 +1,11 @@
+This conversion filter is dedicated to convert from/to different units.
```

Comment:
Is it dedicated to any unit conversions or just distance? Maybe it would make sense to rename if it's only for distance.